x-pack/plugins/security_solution/common/types/timeline/index.ts

```diff
@@ -313,34 +313,36 @@ export type TimelineWithoutExternalRefs = Omit<SavedTimeline, 'dataViewId' | 'sa
  */
 
 export enum TimelineId {
+  active = 'timeline-1',
+  casePage = 'timeline-case',
```

Comment:
I ran into this too with the alert details page. A future step would be separating the alert details flyout state from timeline since that's the only reason the case page even has this. We can prob add it to the backlog